### PR TITLE
PR preparation

### PR DESCRIPTION
**Pull request by mmccool**
Following changes made to main document:
- Updated a few stray context URLs, including in directory.tm.json (TDD thing model), validation files, and ontology diagram
     - Removed ed note after directory.tm.json about context URLs needing to be updated
     - Ontology PUML, PNG, and SVG updated
- Used HTTPS whenever possible, e.g. for context URLs
- ednote about SSE Authorization header converted to a note (will appear in PR)

PR Publication prep:
- files copied to publication/5-pr
- small changes to index.html
   - remaining ednotes commented out
   - status changed to "PR"
   - crEnd defined
   - note and link about future changes added to sotd
- static version exported
- HTML tidy applied to get Overview.html using the following (errs.txt also checked in)
  ```
  tidy -f errs.txt -ashtml --drop-empty-elements no static.html > Overview.html
  ```

TO DO:
- pub rules checker
- link checker 
- diff

***
<a href="https://pr-preview.s3.amazonaws.com/mmccool/wot-discovery/pull/501.html" title="Last updated on Jun 9, 2023, 3:10 PM UTC (6db50dd)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-discovery/501/e07f475...mmccool:6db50dd.html" title="Last updated on Jun 9, 2023, 3:10 PM UTC (6db50dd)">Diff</a>

### CODE REVIEW

**Comment by mmccool** [comment]
Statically was not cooperating, so had to merge this in order to use tools like linkchecker.  Will confirm changes to main index.html content (e.g. URL fixes) before publication.

File: index.html [hunk 1]
```diff
@@ -91,7 +102,7 @@
             },
 	    "REST-IOT" : {
                   title: "RESTful Design for Internet of Things Systems"
-                , href: "https://tools.ietf.org/html/draft-irtf-t2trg-rest-iot-06"
+                , href: "https://datatracker.ietf.org/doc/html/draft-irtf-t2trg-rest-iot-06"
```
Comment (k-toumura) [inline index.html:105]: Should we refer [the latest draft](https://datatracker.ietf.org/doc/html/draft-irtf-t2trg-rest-iot-11)?
The title is also changed to "Guidance on RESTful Design for Internet of Things Systems" from draft-08.

File: index.html [hunk 2]
```diff
@@ -3015,14 +3023,14 @@ <h2>Incremental Transfer</h2>
                 <li>
 		<!-- span class="not-a-rfc2119-assertion" id="perf-inc-chunked-1" -->
                 HTTP/1.1 servers should support `chunked` 
-                <a href="https://tools.ietf.org/html/rfc7230#section-3.3.1">Transfer-Encoding</a> [[RFC7230]]
+                <a href="https://datatracker.ietf.org/doc/html/rfc7230#section-3.3.1">Transfer-Encoding</a> [[RFC7230]]
```
Comment (k-toumura) [inline index.html:3026]: RFC7230 is obsoleted by RFC9112.
```
<a href="https://datatracker.ietf.org/doc/html/rfc9112#section-6.1">Transfer-Encoding</a> [[RFC9112]]
```

File: index.html [hunk 3]
```diff
@@ -3015,14 +3023,14 @@ <h2>Incremental Transfer</h2>
...
                 to receive and server data incrementally.
 		<!-- /span -->
                 </li>
                 <li>
 		<!-- span class="not-a-rfc2119-assertion" id="perf-inc-chunked-2" -->
                 HTTP/2 servers should handle the data incrementally using 
-                <a href="https://tools.ietf.org/html/rfc7540#section-4">HTTP Frames</a> [[RFC7540]].
+                <a href="https://datatracker.ietf.org/doc/html/rfc7540#section-4">HTTP Frames</a> [[RFC7540]].
```
Comment (k-toumura) [inline index.html:3033]: RFC7540 is obsoleted by RFC9113.
```
<a href="https://datatracker.ietf.org/doc/html/rfc9113#name-http-frames">HTTP Frames</a> [[RFC9113]].
```

**Comment by k-toumura** [comment]
Also, RFC7231 is obsoleted by RFC9110.

**Pull request merged**